src/components/app/pageReferrals/yourDistrictRank.tsx

```diff
@@ -63,18 +89,18 @@ function YourDistrictRankContent(props: YourDistrictRankContentProps) {
     )
   }
 
-  const { data } = districtRankingResponse
   if (!data) {
+    let message = 'District rank not found, please try a different address.'
+
+    if (!isUSAddress) {
+      message =
+        "Looks like your address is outside the U.S., so it's not part of any district here."
+    }
```

Comment:
np: this could be a ternary (maybe even inline but I'm ok with separating from render)
```suggestion
    const invalidAddressErrorMessage = isUSAddress
      ? 'District rank not found, please try a different address.'
      :  "Looks like your address is outside the U.S., so it's not part of any district here."
```